### DISCUSSION TITLE
Links in code blocks

### DISCUSSION THREAD

**Discussion by elementalTIMING** [squidfunk/mkdocs-material#5559] [Ideas]
Hi,

is it or would it be possible to have links in code blocks?

Thx and best regards,
Lars

**Comment by squidfunk** [comment]
In general, no, but I think MkDocstrings has this functionality, cc @pawamoy

**Reply by pawamoy** [reply]
Yes, the Python handler of mkdocstrings has an insiders feature to render links to other objects (cross-references) in signature code blocks. In the future we might also have links in source code blocks, for every known name/symbol.

But we do not provide any way to manually render links in code blocks if that's what you're looking for. For this you can write HTML: `<code>` tags can contain `<a>` ones.

**Reply by elementalTIMING** [reply]
Thx a lot for the hint. I'll give it a try!
